src/net.rs

```diff
@@ -265,6 +276,10 @@ pub struct SignatureShareResponse {
     pub signer_id: u32,
     /// Signature shares from this Signer
     pub signature_shares: Vec<SignatureShare>,
+    /// Whether to make a taproot signature
+    pub is_taproot: bool,
```

Comment:
SignatureShareResponse still has the taproot flags but isn't hashing them

Reply:
It is also not needed anymore! Removed :D